app/models/guest.rb

```diff
@@ -0,0 +1,8 @@
+class Guest < ApplicationRecord
+  has_many :guest_rooms
+  has_many :rooms, through: :guest_rooms
+
+  def self.count_guests
+    count(:id)
```

Comment:
This method works, but you will want ot be careful how this method is both tested & implemented, since hte user story is asking you to find `the number of guests that have stayed in that room.`. Since you created a class method, you will only want to call this on a Room instance's Guests collection. 

This would be a little easier to test and implement if it were an instance method on Room, that used the `guests` association.